### DISCUSSION TITLE
Not recognizing SD card on initial powerup

### DISCUSSION THREAD

**Discussion by primordiality** [erichelgeson/BlueSCSI-v1#21] [Q&A]
Hello,

Having an odd issue with my bluepill. When I initially turn on the computer, the LED flashes 5 times to indicate there's no SD card, even though one is inserted. If I push reset on the bluepill, it immediately springs to life and functions normally. It continues to function fine through reboots, but once power is lost from turning off the computer, it will not work without pushing reset.

This is on a Mac SE and was self-assembled. Any help appreciated.

Thank you.

**Comment by marius137** [comment]
Going through the obvious things: did you double check your solder joints for bridges/dry joints?
Did you try a different micro SD?

**Reply by primordiality** [reply]
Yes, and I tried my SD card from my other blue pill, same issue.

**Reply by primordiality** [reply]
Built a second one, same problem, but hooking up USB power to it has it running fine. I am wondering if I've done something wrong, since the pre-built one works without external power.

**Reply by erichelgeson** [reply]
Could you post some photos close up of your work? maybe something will stand out on what to retouch.

**Reply by primordiality** [reply]
![Photo May 03, 20 25 00](https://user-images.githubusercontent.com/4105924/116957926-f6529a80-ac4d-11eb-81b4-f1a7ebeb085e.jpg)
![Photo May 03, 20 24 47](https://user-images.githubusercontent.com/4105924/116957933-f783c780-ac4d-11eb-8176-dcfee65a0c02.jpg)

**Reply by erichelgeson** [reply]
Those all look pretty good. I know it's a bit silly but could backup your sd card and use this utility to format your card, restore the files and try again? https://www.sdcard.org/downloads/formatter/

I've had a few people resolve odd issue just by doing this, so cant hurt.

**Reply by marius137** [reply]
have you tried installing the bluepill from the working board in your board? maybe your bluepills are a bit iffy

**Reply by primordiality** [reply]
@erichelgeson Sorry for the long delay, end-of-the-school year is busy. I tried the SD Card formatter you suggested, still no luck. I thought providing extra power from the 4-pin connector would help too, but, it worked once, otherwise, I still need to hit the reset button on the BluePill for it to recognize the card or have power from the USB port on the bluepill directly before starting the mac.
